go.mod

```diff
@@ -40,6 +40,3 @@ replace github.com/cosmos/ibc-go => github.com/crypto-org-chain/ibc-go v1.0.1-ho
 replace github.com/peggyjv/gravity-bridge/module => github.com/crypto-org-chain/gravity-bridge/module v0.1.22-0.20211004080324-36ed315b3281
 
 replace github.com/cosmos/iavl => github.com/cosmos/iavl v0.17.1
```

Comment:
is this IAVL line necessary? it should be by default in 0.44.1?

Reply:
https://github.com/tendermint/tendermint/blob/v0.34.13/go.mod#L16

it seems still needed, tendermint depends on the old iavl

Reply:
submitted https://github.com/tendermint/tendermint/issues/7089, I think it can be backported to v0.34.x.